qa/packaging/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -1126,25 +1124,25 @@ public void test171AdditionalCliOptionsAreForwarded() throws Exception {
     }
 
     /**
-     * Check that the UBI images has the correct license information in the correct place.
+     * Check that the Docker images have the correct license information in the correct place.
      */
-    public void test200UbiImagesHaveLicenseDirectory() {
-        assumeTrue(distribution.packaging == Packaging.DOCKER_UBI);
+    public void test200ImagesHaveLicenseDirectory() {
+        assumeTrue(distribution.packaging != Packaging.DOCKER_IRON_BANK);
```

Comment:
these tests haven't run before for wolfi or cloud-less right? If they now pass also for those even better

Reply:
Yes. They should run for more images now. This is also the reason I've changed the Dockerfile to add license for all images.